src/api/EscargotPublic.h

```diff
@@ -1440,6 +1440,7 @@ class ESCARGOT_EXPORT ArrayBufferObjectRef : public ObjectRef {
     static ArrayBufferObjectRef* create(ExecutionStateRef* state);
     void allocateBuffer(ExecutionStateRef* state, size_t bytelength);
     void attachBuffer(BackingStoreRef* backingStore);
+    void attachSharedBuffer(BackingStoreRef* backingStore);
```

Comment:
Could you add cctest related to this?

Reply:
IMO we would not add this kinds of API
we can track what the buffer is shared by refcount

Reply:
Thanks for your reviews.
I revert it back to the origin.
(shared buffer would be finally released, but it might need some time for gc to remove it)